rotkehlchen/tests/api/test_history.py

```diff
@@ -337,9 +344,19 @@ def test_query_pnl_report_events_pagination_filtering(
             assert x['timestamp'] >= events[idx + 1]['timestamp']
 
 
-@pytest.mark.parametrize('ethereum_accounts', [])
-def test_history_debug_export(rotkehlchen_api_server):
+@pytest.mark.parametrize('ethereum_accounts', [[]])
+@pytest.mark.parametrize('have_decoders', [[True]])
+def test_history_debug_export(rotkehlchen_api_server: 'APIServer') -> None:
```

Comment:
So this test was not running. Cool. We need to check if more tests are skipped due to this or any other weird unexpected conditions.